pages/404.tsx

```diff
@@ -0,0 +1,42 @@
+import Wrapper from "@components/SDCWrapper";
+import SvgErrorIcon from "@components/icons/ErrorIcon";
+import { Button, Box, Center, Heading, Text } from "@chakra-ui/react";
+
+type ComponentProps = {
+    session: Record<string, unknown>;
+};
+
+export default function Custom404(props: ComponentProps): JSX.Element {
+    const goToHome = () => {
+        window.location.href = "/";
+    };
+
+    return (
+        <Wrapper session={props.session}>
+            <Center minHeight="85vh" align="center">
+                <Box align="center" width="70%">
+                    <SvgErrorIcon />
+                    <Heading size="md" margin="3% 0% 3% 0%">
+                        Oh no! Page not found.
+                    </Heading>
+                    <Text size="md" marginBottom="3%">
```

Comment:
Same as first comment. Chakra shortcut is nice. 
```suggestion
                    <Text size="md" mb={3}>
```